src/sentry/models/auditlogentry.py

```diff
@@ -243,4 +246,7 @@ def get_note(self):
         elif self.event == AuditLogEntryEvent.RULE_REMOVE:
             return 'removed rule "%s"' % (self.data['label'], )
 
+        elif self.event == AuditLogEntryEvent.SET_ONDEMAND:
+            return 'changed on-demand max spend to $%d' % (self.data['ondemand'] / 100, )
```

Comment:
Note that this is going to only show integer values (just dollars), not decimals. I'm not sure if that's relevant.

Reply:
On-demand budgets can only be in dollars. Because we're evil, greedy people 🤑